core/src/main/java/com/novoda/landingstrip/ScrollOffsetCalculator.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.landingstrip;
+
+import android.view.View;
+
+class ScrollOffsetCalculator {
+
+    private final TabsContainer tabsContainer;
+
+    ScrollOffsetCalculator(TabsContainer tabsContainer) {
+        this.tabsContainer = tabsContainer;
+    }
+
+    int calculateScrollOffset(int position, float pagerOffset) {
+        View tabForPosition = tabsContainer.getTabAt(position);
+
+        float tabStartX = tabForPosition.getLeft() + getHorizontalScrollOffset(position, pagerOffset);
+
+        int viewMiddleOffset = getTabParentWidth() / 2;
```

Comment:
`* 0.5f` here but not on the next line? The offset should be in pixels too right, not semiPixels